### DISCUSSION TITLE
Module 8 docker containers not appearing in my codespace

### DISCUSSION THREAD

**Discussion by wtb4rtr** [ua-gist604b-f24/syllabus#41] [General]
Hi I'm starting Module 8; the instructions say something to the effect of we will be using a docker container which has the pre-installed libraries we need (presumably gdal).  I created the codespace and there are no docker containers; are we supposed to create them or are they supposed to be pre-installed like in the previous Modules?  Thanks in advance and good luck to everyone with the end of the quarter
Bill Brown

**Comment by kev-bone** [comment]
Once my codespace started up, I was able to immediately begin running GDAL commands.

**Comment by wtb4rtr** [comment]
I ran the first command and was given the message that the command was not recognized; thanks for the help

**Comment by wtb4rtr** [comment]
Hey Kevin I stand corrected; I tried it again and it worked; I guess I had a typo.  Thank you for your help and happy Thanksgiving
Bill Brown
